Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -493,6 +507,42 @@ static void __GetGlyphRunDetails(_CTTypesetter* ts, CFRange range, CGRect frameS
                 line->_width += glyphRunDetails._dwriteGlyphRun[i].glyphAdvances[index];
             }
 
+            DWRITE_GLYPH_METRICS glyphMetrics[run->_dwriteGlyphRun.glyphCount];
```

Comment:
We should be able to get the frame bounds directly from dwrite instead of doing an O(N) operation like we do here.  How did that work before this change anyway?

---
In reply to: [80313171](https://github.com/Microsoft/WinObjC/pull/1027#discussion_r80313171) [](ancestors = 80313171)